TABTestKit/Classes/BDD/And.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  And.swift
+//  TABTestKit
+//
+//  Created by Stefan Renne on 09/11/2020.
+//
+
+import XCTest
+
+/// Defines an executable And step to be used in tests.
+///
+extension XCTestCase {
```

Comment:
What's the thought process behind removing the typealiases and creating a bunch of extra functions?

Reply:
The ability to detect if it's a `Given`, `When`, `Then` or `And` while still having access to the XCTestCase

Reply:
Fixed this together with the TABTestCase.current change